### ISSUE TITLE
Arbitrary snakemake parameters do not work

### ISSUE THREAD

**Issue by mmcguffi** [Wytamma/snk#42]
First, thank you so much for making this -- this is an incredible project and Im shocked that it's so under-the-radar right now. Really impressive work!

I cant seem to get `snk` to take arbitrary (and often important) snakemake command line parameters. For example:

```bash
variant-calling run -r .test/config -r .test/data -n --cores 20 --keep-going 

╭─ Error ──────────────────────────────────────────────────────────────────────────────────────────────────╮
│ No such command '--keep-going'.                                                                          │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

Re-arrange parameters:
```bash
variant-calling run -r .test/config -r .test/data --cores 20 --keep-going -n
╭─ Error ──────────────────────────────────────────────────────────────────────────────────────────────────╮
│ No such command '-n'.                                                                                    │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

Perhaps Im just missing something? Any help / tips would be appreciated :)

**Comment by Wytamma** [comment]
Hey @mmcguffi! Thanks for the kind words :) I've fixed this bug in v0.13.0. I also added the ability to pass snakemake options that are masked by the CLI to snakemake by adding the `--snake` prefix to the option. 

e.g. `--snake-r` will use the snakmake reason option `-r` not the `snk` resource option `-r`

**Comment by mmcguffi** [comment]
Seems to work perfectly! Thank you for the (insanely fast) fix :)
